src/WebJobs.Extensions.DurableTask/Listener/TaskEntityShim.cs

```diff
@@ -755,7 +755,7 @@ private async Task ExecuteOutOfProcBatch()
                         request.IsSignal,
                         request.ScheduledTime,
                         parentTraceContext,
-                        request.RequestTime.Value);
+                        request.RequestTime);
```

Comment:
Why do we need this change?

Reply:
This is due to having to make the `startTime` parameter in the TraceHelper method signature a `DateTimeOffset?` rather than `DateTimeOffset` as discussed in our convo [above](https://github.com/Azure/azure-functions-durable-extension/pull/3076#discussion_r2092073142), and so we no longer have to extract the `Value` here and can just directly pass in `request.RequestTime` (which has type `DateTimeOffset?`)